pkg/arvo/app/landscape/index.html

```diff
@@ -21,6 +21,16 @@
   </head>
   <body>
     <div id="root"/>
+    <script src="/~wallet/js/bcoin.js"></script>
```

Comment:
Note: these scripts should just be included in the main App bundle, not loaded separately

Reply:
I'll look into it but this has been lingering since the old build flow with gulp. I couldn't get that to work so adding them separately was my only successful attempt, but maybe it'll be a bit easier with webpack. fyi, these files were created with a [bundler](https://github.com/chjj/bpkg) used by bcoin so there might be some issues integrating it with webpack.